docs/System_toolchain.rst

```diff
@@ -0,0 +1,148 @@
+
+  WARNING: Deprecated functionality, will no longer work in v5.0:
+  Use of 'dummy' toolchain is deprecated, use 'system' toolchain instead;
+  see http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.html for more information
+
+
+Other than this warning, **there is no impact on the EasyBuild functionality whatsoever**.
+
+Use of the ``dummy`` toolchain has been deprecated, but is still supported in EasyBuild v4.x; it is simply
+an alias for the ``system`` toolchain which replaces it.
+
+Hence, software installations for easyconfig files that are still using the ``dummy`` toolchain will proceed as before,
+and the contents of the easyconfig files will be interpreted exactly as before, with one exception:
+the (build) dependencies specified in the easyconfig files will *always* be loaded in the build environment,
+*regardless* of the toolchain version (see also :ref:`system_toolchain_motivation_deprecating_dummy_version_and_deps`).
+
+Since the ``dummy`` toolchain has been deprecated, so has the ``--add-dummy-to-minimal-toolchains`` configuration option.
+It has been replaced with the ``--add-system-to-minimal-toolchains`` configuration option (see also :ref:`minimal_toolchains_system`).
+
+
+.. _system_toolchain_updating_dummy:
+
+Updating easyconfig files that use the ``dummy`` toolchain
+--------------------------------------------------------
+
+To update easyconfig files that still use the ``dummy`` toolchain, you should simply change the definition
+of the ``toolchain`` parameter to use ``'system'`` as a toolchain name (see also :ref:`system_toolchain_usage`).
+
+We recommend using the ``SYSTEM`` constant for this, see :ref:`system_toolchain_SYSTEM_constant`.
+
+To automatically update easyconfig files still using the ``dummy`` toolchain, you can also use
+``eb --fix-deprecated-easyconfigs``. Do take into account that this option could also result in
+additional changes being made to the easyconfig files,
+see :ref:`easyconfig_files_local_variables_fix_deprecated_easyconfigs`.
+
+
+.. _system_toolchain_motivation_deprecating_dummy:
+
+Motivation for deprecating the ``dummy`` toolchain
+--------------------------------------------------
+
+Deprecating the ``dummy`` toolchain was done for a number of reasons:
+
+* :ref:`system_toolchain_motivation_deprecating_dummy_name`
+* :ref:`system_toolchain_motivation_deprecating_dummy_version_and_deps`
+
+.. _system_toolchain_motivation_deprecating_dummy_name:
+
+``dummy`` toolchain name
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``dummy`` name may be rather confusing, especially if you are new to EasyBuild.
+Just based on the name, it is unclear how the ``dummy`` toolchain is different from a regular toolchain.
+
+Hopefully, the name of the ``system`` toolchain does speak more for itself: when using it, you are relying on the
+compiler & lilbraries provided by the operating system (as opposed to compilers & libraries that are a part of a regular toolchain).
```

Comment:
"lilbraries" -> "libraries"

Reply:
Hmm, why didn't the spell checker that Travis runs catch this... :-/
Thx!